vaadin-card-flow-parent/vaadin-card-testbench/src/main/java/com/vaadin/flow/component/card/testbench/CardElement.java

```diff
@@ -23,4 +28,96 @@
  */
 @Element("vaadin-card")
 public class CardElement extends TestBenchElement {
+
+    /**
+     * Gets the content elements.
+     *
+     * @return the content elements
+     */
+    public List<TestBenchElement> getContentElements() {
```

Comment:
Since all other methods (and other TB classes) don't use the "Element" suffix, maybe this should just be `getContents`? For the footer it could be `getFooterContents`.

Reply:
Done.